### ISSUE TITLE
The Bactrian has too much cargo space and too many bunks.

### ISSUE THREAD

**Issue by quyykk** [endless-sky/endless-sky#6073]
The Bactrian is classified under "Heavy Warships" because well, it is a heavy warship. It has comparable shield/hull values to other human heavy warships and comparable weapon/engine space. The only extra it has are the cargo space and the bunks.

Now the problem is that it just has too many. From all heavy warships you can buy, the second in terms of cargo space is the Hai heavy warships that have 142. Compared to the 530 of the Bactrian, that's nothing. The cargo space of the Bactrian is comparable to Human Heavy Freighters.

It also has way too many bunks (enough for 171 extra passengers). In comparison, the best human Transport ship is the Centipede with 161 free bunks. The only transport ship you can buy with more is the Wanderer Riptide and that one only has 185 extra bunks.

IMO for me at least, it isn't a problem that the Bactrian is a jack-of-all-trades, but rather that it is the if not the second best as transport and freighter even though it's only a heavy warship. Yes, I know that the Bactrian is locked behind a mission and that it's pretty expensive. But it still is a ship that rivals the best transport, heavy freighter and heavy warships of the Humans. It's basically a no-brainer once you unlock it (and if you can live with its low shields/hull, than it's one of the best ships in the game due to the generous outfit, weapon and engine space - which can be easily extended because of the large cargo space).

I think the cargo space and the bunks should be significantly nerfed.

There are been similar discussions in the past ([this comment is relevant](https://github.com/endless-sky/endless-sky/pull/2076#issuecomment-286856014) and [this Steam discussion](https://steamcommunity.com/app/404410/discussions/2/143388132200579499/)), but I couldn't find any newer talks/proposals.

**Comment by Zitchas** [comment]
Within human space the Bactrian is:
The best transport (the Auxillary has 5 more bunks, but 17 more crew, so Bactrian has more free bunks)
The second best freighter (only 70 behind the bulk freighter, and 40 ahead of the Behemoth, and 140 ahead of the hauler III)

As a warship, the Bactrian has 700 more health than the Dreadnought (supposedly high tech, brand new, top of the line FW heavy warship) and 100 more health than the Cruiser. It has more turrets than any other human ship, and has 3 fighters and 4 guns on top of that, which gives it another edge against the Dreadnought too. (which has no carrier capabilities.

My objection to the Bactrian basically stems from its labeling: It is a city ship. This is supposed to be a ship that was sent out into unexplored, uninhabited areas to setup colonies. Uninhabited areas by definition have nothing for pirates to prey on, so they are comparatively safe. As per the description, having a huge bunk and cargo capacity makes sense: Setting up a colony takes a lot of people and a lot of supplies. What it *doesn't* take is large amounts of armaments. 

Now, the Bactrian occupies a neat spot as being one of the few (and best) jack of all trades ships in the game. I don't mind that, but it needs to have some weaknesses so that it isn't the be-all and end-all of human ships. Based on its lore, its name, and its location (The Deep is the safest and least-pirate infested part of human space), I would suggest removing all the gun hardpoints and about a quarter of its shields. It could even stand to gain 15-20 more bunks. I would also swap it to being called a Transport instead of Heavy Warship. I find it a lot more interesting to think of it as the toughest transport, rather than the heavy warship that can haul the most people.

It would still be tougher than any of the "non-licensed" heavy warships, and definitely still king of the looter economy, but it would no longer overshadow the dedicated combat ships in the storylines. It would also make things a bit more interesting because it would establish that story-line unlocks do not necessarily always have to be better *warships.* This would move it firmly into the realm of "toughest of the merchant ships" rather than "Best Unquestioned human ship". MZ has long had the philosophy for outfits that there shouldn't be any "best" outfits, just best at specific situations. It makes sense that we would do the same thing with ships, and the Bactrian stands at the forefront of that.

**Comment by gunqqer** [comment]
Didn't we already address the bactrain in #4253, #4387, and most importantly #3120 ? I feel like it is in a good place right now and if anything we should be having more ships of the same caliber instead of nerfing it again.

**Comment by Zitchas** [comment]
No, I don't think it is. Better, than it was, though. It still doesn't match its lore.

**Comment by quyykk** [comment]
What I don't like is that the Bactrian is currently the best ship in Human space. Every other Human ship is inferior. If you have enough money, why would you buy any other ship?

- Need a transport ship? The Bactrian has more than any other transport ship.
- Need a heavy freighter? The Bactrian has just a tiny bit less cargo space than the best heavy freighter and is way faster and has more weapon space.
- Need a heavy warship? The Bactrian has comparable engine/weapon space and gun/turret ports than the best heavy warships.

**Comment by MCOfficer** [comment]
Numbers are based on [derpy's spreadsheets](https://docs.google.com/spreadsheets/d/1ZGW3Mtc4ylsSBknupPCID0u9waiFeXI3AIvYwrfGhF8/).

> Need a transport ship? The Bactrian has more than any other transport ship.

It outclasses the stock star queen as much as it is outclassed by the auxiliary. If you minmax the star queen, you can get comparable numbers out of a civilian ship that's available without mission.

> Need a heavy freighter? The Bactrian has just a tiny bit less cargo space than the best heavy freighter and is way faster and has more weapon space.

behemoth 490, bactrian 530, bulk freighter 600.

> Need a heavy warship? The Bactrian has comparable engine/weapon space and gun/turret ports than the best heavy warships.

I assume the main contenders we should pit the Bactrian against are the carrier and dreadnought. (Arguably the Kestrel too).

In that regard, the Bactrian loses out outfit-, engine- and weapon space. It beats the dreadnought in fighter bays and turret mounts, but has inferior gun ports. And the amount of weapon space you have means that you can never have all ports equipped with top-of-the-line weaponry, you're gonna have to make compromises.

---

bottom line, the stock bactrian is definitely in contention for the top spot for all categories. You can minmax it to the definitive top in some of them (transport and cargo), at the expense of it acting as jack-of-all-trades.

**Comment by guinimesa** [comment]
The Bactrian is strong and easily the best Human ship however it's use later on is basically a raiding/boarding ship. It is very big so it can't contend with stronger and faster ships, the hull is comparable to a Shield Beetle but then shields are half, drag is almost twice and also has 300 extra mass and let's not even compare it to the Wanderer's Derecho or any other because they lose in every aspect except cargo space and bunks.
The Bactrian is perfectly made to be an end game boarding/trading ship but not a dedicated combat ship and while it can outperform any other human ship it still is barely enough for more than that. Oh yeah, it's also locked behind a long and bothersome mission so you won't get it easily.
If you think lore-wise then having many weapons for a flying haven or city-ship then it is perfectly correct, you just don't send such a huge ship without protection and sending more ships for the same is not worth it so you send 1 big and strong ship.
I do agree that some changes could be made, increasing the size to properly become a city ship for example but after that I would believe that it is balanced.

**Comment by Gen0** [comment]
The Bac is the evolution of the Mule which is an evolution of the Argosy. Each are 'Multirole' ships and ideal flagships at different stages. Ideal because the cargo space is a boon for larger raids where you can't invade everything but you can pick the spoils for parts when your crew is low but there are still ships to loot.

It is a city ship but calling it one would make the mule a township and the argo a village ship and that's just silly. It's a multirole carrier but this game doesn't separate carriers, capitals and heavy fighters because there's not that many of each per station.

**Comment by ThrosturX** [comment]
How about considering splitting the Bactrian up into two variants, one of which will retain warship capabilities in exchange for some bunks and cargo space while the other loses the fighter bays and retains only a modest weapon layout (something like 2 or 3 turrets and maybe two gun slots for missiles or something, idk). I think doing this coupled with a balance pass (boost) of the more advanced human ships like the auxiliary and dreadnaught could go a long way in making the Bactrian less of a no-brainer and more of a conscious choice.

In a recent play-through, I really enjoyed having to first capture a shield beetle so that I could get my first raider and jump drive, it was a fun challenge and I _really_ had to gimp the beetle in every way to get it up to par to capture a raider. In my first playthrough, I captured raiders for breakfast in a tricked out Bactrian that still had plenty of weapons installed. It was ridiculously easy to kill and capture the raider with a single ship (comparatively speaking -- it wasn't easy per-se, but it was much much easier than using any other ship that you can acquire so early in the game).

I also remember in my first playthrough, that by the time I could have access to the nicer navy ships like the carrier and auxiliary, there was no reason for me to use them, due to the Bactrian just being more customizable. This brings me back to my original point, the Bactrian is **too** customizable, but splitting it up into two variants (the "City Ship" variant and the "Protector" variant), the player will still be able to do the same set of things with the two variants, but it would be unrealistic to solo a raider and then capture it in the same ship (as it should be!).

**Comment by VonZorch** [comment]
> My objection to the Bactrian basically stems from its labeling: It is a city ship. This is supposed to be a ship that was sent out into unexplored, uninhabited areas to setup colonies. Uninhabited areas by definition have nothing for pirates to prey on, so they are comparatively safe.

The important word here is unexplored.  If you're going haring off into the great beyond, without support, you need a heavily armed ship in case you run into hostile strangers.

**Comment by Zitchas** [comment]
> 
> 
> > My objection to the Bactrian basically stems from its labeling: It is a city ship. This is supposed to be a ship that was sent out into unexplored, uninhabited areas to setup colonies. Uninhabited areas by definition have nothing for pirates to prey on, so they are comparatively safe.
> 
> The important word here is unexplored. If you're going haring off into the great beyond, without support, you need a heavily armed ship in case you run into hostile strangers.

There are really only two times that it makes sense for exploration ships to be heavily armed:
1) When you have infinite resources and equipping them like warships doesn't impact their functionality as colonization/exploration ships. (Star Trek is a perfect example of this)
2) When you are exploring areas that are known to have (or very likely to have) hostiles. Ex. military scouts doing recon, and "exploring" the fringes of settled space occupied by prospectors, anarchists, and others and thus where pirates have a reason to be. (I put "exploring" in quotes like that because it isn't actually exploring - people are already there - it is more just documenting things that other people have already found. At most, it is the first *official* exploration of the area)

It is important to define what kind of "unexplored" we are talking about. If we are talking about the fringes of settled space, like the next system over from inhabited worlds, that's one thing. These areas are only unexplored in the sense that no-one official has seen fit to map them yet. There is a very high likelyhood that such areas have already been explored by prospectors, miners, people wanting to stake their own claim away from everyone else, criminals establishing hideouts away from civilization, etc. And of course with all those people running around, pirates have prey to chase, so they'll be out there too. These fringe zones are where the danger is. And these fringe zones can easily be bypassed using interceptors or light warship escorts that escort the Bactrian a couple jumps out to get through the fringe, and then turn around and go back since they don't have the fuel for long trips. This is almost the ideal reason to have interceptors.

On the other hand, the distant unexplored areas are the places where no human has ever gone. And the Bactrian explicitly talks about going places where they are weeks away from the nearest place they can refuel. There's no pirates out there. There's *no one* out there. The one alien that humans may have met by the time of colonization with the city-ships is the Quarg, and they are obviously not hostile. And there's a strong chance that humanity hadn't even met them yet. So early city-ships would have faced this choice:
a) Carry along heavy weaponry in case we run into something, and reduce how well we can establish a colony
b) Carry along extra colonization equipment to increase the chances that we can actually establish a successful colony.

In real-world terms, this exploration could be modeled thus:
- "Wild West Frontiers" exploration.
- "First Expedition to Antarctica" exploration.

Wild West frontiers is the sort of exploration that takes place on the fringes, there's regular flows of prospectors and the like going into and out of it. There's people, there's crime, and theres a rapidly changing situation. This is the area within a couple jumps of colonized planets. This is the person versus person environment we see a lot of in the movies and TV shows. The Firefly series is all about running around in this transition zone, bouncing between sparsely settled areas. And worth noting that, despite ever present threats, the Serenity carried no heavy weaponry of any kind. Not even an anti-missile turret. The one time in the TV series that they shot something in space they used Jayne's rifle. This environment is perfectly suited to midrange transports and freighters. Small and medium ships can handle things just fine, anything bigger is probably overkill for trading with a small settlement. 

First Expedition to Antarctica is the voyages into the deep unknown. There's no one there. These are people who are quite possibly the first humans ever to reach these places. And as far as humans in the Pre-Quarg era are concerned, there's no such thing as aliens at all. This is purely the realm of person versus environment. And guns don't work on the environment, and it *will* kill people even more surely than any other threat will. One can run away from a person, or an alien, or a predator. But hunger *will* always catch up in the end. Cold and exposure are even deadlier. And it is this environment that the city ships were built to operate in, so I would expect them to be optimized for it. While a small settlement will have nowhere near enough to trade to make it worth while for a heavy freighter or transport, *establishing* a new settlement will tax such massive ships to the limit. That's why city-ships are a thing, really. To be specialists in establishing new settlements out of nothing.

Establishing a colony is *hard*. It is the sort of thing that would take up every single last ton of space available on a cityship. And even then they'll stuff the cabins with stuff and line the hallways too. As in, strip a Bactrian of *everything,* downgrade the engines to the smallest available, and then max out on a mix of cargo expansions and some bunks. *maybe* give it a single anti-missile, but that's mostly for shooting problematic asteroids, not combat.

In the era that cityships were built, (and, for that matter, in the present day of Endless Sky) humanity does *not* have access to infinite resources, particularly in terms of space available on ships. For every missile it carries, that's a week less of supplies that it could carry for prospective colonists. And that means that putting a single torpedo launcher on the cityship could easily cost them enough resources that the colonization effort fails. 

Putting heavy weaponry on a city-ship makes as much sense as strapping naval artillery to SpaceX's starship in fear that it will run into hostile aliens on Mars. They could do it, but it would probably jeapordize the mission to do so. If the ship captain is *really* scared that they are going to run into hostiles at their destination, they could equip their ship with bigger engines so they can run away. At least those will serve double purpose in getting them to their destination faster. But heavy weaponry? Total waste of space.

This isn't Star Trek. Especially during the early days when humanity was establishing the first few colonies, the galaxy was a vast, empty place. As far as humanity could tell, they were alone in the universe. There was no first contact with *anyone*. (recall that in Star Trek, humanity met aliens after their first successful warp drive experiment. So they knew aliens were out there before they even left the solar system. And the vulcans had contact with other races, I think.) Engines were weak, power sources poor, and every pound counted. And even in the present day in ES, we still aren't anywhere close to where Star Trek is in the original series. There's no infinite power systems, no teleportation, and definitely no replicators. Believe me, if the enterprise had to carry enough resources to last even a single year of its original five year mission instead of replicating things from their basically infinite energy. It'd probably have a tenth as many people, no torpedoes, and most of its onboard space would be taken up by cargo bays full of supplies. Vast hydroponics farms are also a possibility.

**Comment by Quantumshark** [comment]
Is this still a problem? The Bactrian's bunks and cargo are both eclipsed by the Rano'erek, and it has gained a clear weakness of poor mobility as a result of #9616. Feel free to reopen this if you disagree, but in the current state of the game I think the Bactrian is balanced.

**Comment by Zitchas** [comment]
Seems reasonable to close this. It's in a much different place now. 

Likewise, I think the general accepted lore is that the Bactrian isn't (and never was) the best or biggest of the City Ships, it was, however, the best *combat* city ship, especially with modern upgrades. As such, presumably its specialty back in the day would have been supporting colonization efforts in the fringe regions where pirates, anarchists, and other potential threats would be common. (and, as described prior, more of a "Wild West" exploration rather than "Voyage into the Unknown" exploration.)

So, agreed with this being closed. I'm sure it will be subject to further revisions in the future, but this particular one is probably good to close.
